source/integrate/github-interoperability.rst

```diff
@@ -102,6 +102,11 @@ Upgrade
 
 We recommend updating this integration as new versions are released. Generally, updates are seamless and don't interrupt the user experience in Mattermost. Visit the `Releases page <https://github.com/mattermost/mattermost-plugin-github/releases>`__ for the latest release, available releases, and compatibiilty considerations.
 
+.. important::
+
+  - We strongly recommend removing older versions of this integration before uploading a new version. Mattermost server selects the newest integration version automatically, unless :ref:`automatic prepackaged plugins <configure/plugins-configuration-settings:automatic prepackaged plugins>` is disabled because precise control over plugin versions is preferred.
+  - Use the System Console, REST API, or in-product Marketplace to manage this integration. Manually placing Mattermost integration files in the ``/plugins`` directory isn't recommended and will be `deprecated in future versions <https://forum.mattermost.com/t/deprecation-notice-manual-plugin-deployment/21192>`_.
+
```

Comment:
From here on out, I'd propose reverting all the changes. We don't need to duplicate this on a per-plugin basis, IMO.